<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SAG-8JC82X2" ignored="1" name="Loans" tests="16" failures="1" timestamp="2022-04-04T13:39:08 IST" time="0.058" errors="0">
  <testcase name="acceptProv" time="0.004" classname="test.Day1"/>
  <testcase name="acceptProv" time="0.0" classname="test.Day1"/>
  <testcase name="acceptProv" time="0.001" classname="test.Day1"/>
  <testcase name="failedTest" time="0.002" classname="test.Day1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.Day1.failedTest(Day1.java:60)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- failedTest -->
  <testcase name="grop1" time="0.001" classname="test.Day1"/>
  <testcase name="dep" time="0.001" classname="test.Day1"/>
  <testcase name="pTest" time="0.0" classname="test.Day1"/>
  <testcase name="calculateHomeloan" time="0.0" classname="test.Day2"/>
  <testcase name="exitHomeloan" time="0.001" classname="test.Day2"/>
  <testcase name="group" time="0.001" classname="test.Day2"/>
  <testcase name="loginAgainloan" time="0.0" classname="test.Day2"/>
  <testcase name="loginHomeloan" time="0.001" classname="test.Day2"/>
  <testcase name="loginNowloan" time="0.0" classname="test.Day2"/>
  <testcase name="exitCarloan" time="0.0" classname="test.Day3"/>
  <testcase name="group" time="0.0" classname="test.Day3"/>
  <testcase name="calculateCarloan" time="0.0" classname="test.Day3"/>
  <testcase name="loginCarloan" time="0.0" classname="test.Day3">
    <ignored/>
  </testcase> <!-- loginCarloan -->
</testsuite> <!-- Loans -->
